components/camera/client.go

```diff
@@ -201,7 +146,7 @@ func (c *client) Stream(
 				return
 			}
 
-			frame, release, err := c.Read(streamCtx)
+			img, err := ImageFromVideoSource(streamCtx, mimeTypeFromCtx, nil, c)
```

Comment:
why are we passing in `mimeTypeFromCtx` and not `ctxWithMIME` here?

Reply:
`ctxWithMIME` is a `Context` and `mimeTypeFromCtx` is a string. Do you mean why we're using `streamCtx` and not `ctxWithMIME`?

Reply:
gothca, I was confused.